airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -208,6 +211,12 @@ def retry_factor(self) -> float:
         """
         return 0 if IS_TESTING else super(StripeStream, self).retry_factor
 
+    def get_cursor(self) -> Optional[Cursor]:
```

Comment:
Nit: We should add a comment that explains that by nulling out the RFR cursors, we're disabling RFR for concurrent CDK because it hasn't been implemented yet.